### PR TITLE
add download button to works page

### PR DESCRIPTION
**Pull request by jamesgorrie**
References #1117 #1004 

## Type

✨ Feature  

- [ ] Demoed to @

## Value
People can download the full image.

- [x] Make download URI Wellcome specific
- [x] Add tracking

### CODE REVIEW

**Comment by jamesgorrie** [comment]
takingyourimage

File: client/js/tracking.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { trackEvent } from './utils/track-event';
 
 export default {
   init: () => {
```
Comment (davidpmccormick) [inline client/js/tracking.js]: Could this be shorthand, just `init() {`?

File: server/views/pages/work.njk [hunk 2]
```diff
@@ -46,6 +46,15 @@
           {% endif %}
 
           {% componentV2 'copy-url', {id: 'work'} %}
+
+          <a href="/download?uri={{ work.imgLink | replace('WIDTH,', 'full') | urlencode }}"
+             download="{{ work.id }}.jpg"
```
Comment (davidpmccormick) [inline server/views/pages/work.njk]: Did you take a look at @gestchild's branch? He notes that

> download attribute won't work across all browsers and when image is from a different origin

File: server/views/pages/work.njk [hunk 3]
```diff
@@ -46,6 +46,15 @@
...
+             rel="noopener noreferrer"
+             class="btn btn--dark {{ {s:4} | spacingClasses({margin: ['top']}) }}"
+             data-track-event='{"category": "works", "action": "download", "label": "{{ work.id }}"}'>
+             {% icon 'actions/download-1' %}
+              Download fullsize image
```
Comment (davidpmccormick) [inline server/views/pages/work.njk]: full size (or at least full-size) rather than fullsize?

**Pull request merged**